### PR TITLE
Fix-Stake-page-UI

### PR DESCRIPTION
**Pull request by ONEONUORA**

### CODE REVIEW

**Comment by djeck1432** [comment]
@ONEONUORA Please, sync with main branch

**Comment by ONEONUORA** [comment]
@djeck1432 I have synced. Thank you

**Review by whateverfw — CHANGES_REQUESTED** [review]
**First of all sync with main one more time.**

Mobile/tablet is broken, please fix

image

File: frontend/src/components/ui/Button/button.css
Comment (whateverfw) [inline frontend/src/components/ui/Button/button.css:1]: please provide correct values for border-radius, according to figma

File: frontend/src/pages/vault/vaultLayout.css
Comment (whateverfw) [inline frontend/src/pages/vault/vaultLayout.css:1]: sidebar is broken now, please merge main, it was fixed in another ticket, make sure that it still fixed in your branch

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Pull request closed without merging**